mapadroid/patcher/add_is_ar_scan_eligible.py

```diff
@@ -0,0 +1,27 @@
+from ._patch_base import PatchBase
+
+
+class Patch(PatchBase):
+    name = 'Add add_is_ar_scan_eligible flag to pokestop and gyms'
+    descr = 'New Protos, more useless data!'
+
+    def _execute(self):
+        alter_pokestop = """
```

Comment:
We should check to see if the column exists prior to executing the sql. This allows the patch to be run multiple times without experience any issues. Here is a good example of that in-use

https://github.com/Map-A-Droid/MAD/blob/master/mapadroid/patcher/pokemon_iv_index.py#L18